### PR TITLE
FAQ - generalize 'beta_ios' for Android/iOS

### PR DESCRIPTION
**Pull request by heinezen**
Fixes https://github.com/corona-warn-app/cwa-website/issues/1305

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -842,12 +842,13 @@
...
-                        "anchor": "beta_ios",
+                        "title": "I am testing a beta version of iOS/Android. What do I have to consider?",
+                        "anchor": "os_beta",
                         "active": true,
                         "textblock": [
-                            "We do not officially support beta versions of the respective operating systems. These are still in development and not intended for productive operation.",
-                            "However, if you as a participant in the Apple beta software programme would like to point out potential errors or incompatibilities at an early stage, <a href='https://github.com/corona-warn-app/cwa-app-ios/issues' target='_blank' rel='noopener noreferrer'>we are happy to receive your feedback on Github</a>."
+                            "Beta versions of the respective operating systems are not officially supported because they are still in development and not intended for productive operation. You can still use the app like normal, but you should be aware that you might encounter errors that are not present when using a release version of your operating system.",
```
Comment (MikeMcC399) [inline src/data/faq.json:849]: "like normal" => "normally"
"release" => "released"

File: src/data/faq.json [hunk 2]
```diff
@@ -842,12 +842,13 @@
...
+                            "However, if you exerience errors, you can feel free to point them or other incompatibilities out by opening an issue on Github:",
```
Comment (MikeMcC399) [inline src/data/faq.json:850]: "exerience" => "experience"
"Github" => "GitHub"

File: src/data/faq_de.json [hunk 3]
```diff
@@ -854,12 +854,13 @@
...
                         "active": true,
                         "textblock": [
-                            "Wir unterstützen offiziell keine Beta-Versionen der jeweiligen Betriebssysteme. Diese sind noch in der Entwicklung und nicht für den produktiven Betrieb vorgesehen.",
-                            "Wenn Sie als Teilnehmer am Apple Beta Software-Programm jedoch frühzeitig auf potentielle Fehler oder Inkompatibilitäten hinweisen möchten, <a href='https://github.com/corona-warn-app/cwa-app-ios/issues' target='_blank' rel='noopener noreferrer'>nehmen wir Ihr Feedback gerne auf Github entgegen</a>."
+                            "Betaversionen der jeweiligen Betriebssysteme werden nicht offiziell unterstützt, da sich die noch in der Entwicklung befinden und nicht für den produktiven Einsatz vorgesehen sind. Sie können die App weiterhin normal benutzen, sollten aber daran denken, dass möglicherweise Fehler auftreten, die in der Releaseversion Ihres Betriebssystems nicht vorhanden sind.",
+                            "Falls bei Ihnen solche Fehler auftreten, können sie diese oder andere Inkompatibilitäten jedoch gerne melden, indem Sie ein Github Issue erstellen:",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:862]: "Github" => "GitHub"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I somehow have the feeling I have made these correctional suggestions before, but maybe I am mistaken?

**Comment by heinezen** [comment]
@MikeMcC399 You're right I forgot that I already posted the entry in the issue.

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json]: fixed

File: src/data/faq.json (hunk 2 above)
Comment (heinezen) [reply src/data/faq.json]: fixed

File: src/data/faq_de.json (hunk 3 above)
Comment (heinezen) [reply src/data/faq_de.json]: fixed

**Pull request merged**